cpp/daal/src/externals/service_spblas.h

```diff
@@ -178,7 +178,7 @@ struct SpBlas
     }
 
 public:
-    static services::Status xsyrk_a_at(const fpType * a, const size_t * ja, const size_t * ia, size_t m, size_t n, fpType * c)
+    static services::Status xsyrk_a_at(const fpType * a, const size_t * ja, const size_t * ia, size_t m, size_t n, fpType * c, const size_t ldC)
     {
         size_t nBlocks = 50;
```

Comment:
Strange hardcode :)

Reply:
I would not touch :) My expectation is to replace it with MKL gemm.